main.py

```diff
@@ -83,6 +89,36 @@
         logger.info("Not recreating sentiment analysis.")
         sentiment_df = FileService.read_parquet_to_df(file_name='articles_sentiment')
 
+    # Embedding modeling
+    top2vec_model_path = "./models/top2vec/labse-three-year"
+    if (
+            not Path(top2vec_model_path).exists()
+            or force_recreate
+            or _previous_step_recreate
+    ):
+        top2vec_model = TopicModelingService.fit_top2vec_model(data_column=sentiment_df["content"])
```

Comment:
Do we maybe want to use the same syntax as in the other services? 

Service.method(df:pd.DataFrame, column_to_process:str="content")?

Reply:
I think for a production ready application standardization would make sense. But for now I would rather not retest. 
Seems like everything is working. 

Also I think in general I would do it the other way around. Pass only the columns the method needs. Experience shows that it is already only one column. Then return only one column.